api/types/access_request_test.go

```diff
@@ -18,12 +18,52 @@ package types
 
 import (
 	"testing"
+	"time"
 
+	"github.com/gravitational/trace"
+	"github.com/jonboulle/clockwork"
 	"github.com/stretchr/testify/require"
+
+	"github.com/gravitational/teleport/api/constants"
 )
 
 func TestAssertAccessRequestImplementsResourceWithLabels(t *testing.T) {
 	ar, err := NewAccessRequest("test", "test", "test")
 	require.NoError(t, err)
 	require.Implements(t, (*ResourceWithLabels)(nil), ar)
 }
+
+func TestValidateAssumeStartTime(t *testing.T) {
+	clock := clockwork.NewFakeClock()
```

Comment:
I don't think you really need a fake clock here, as the test is really about working with arbitrary `time.Time` values.